### PR TITLE
Updating to latest CI configuration.

### PR DESCRIPTION
**Pull request by rayrod2030**

### CODE REVIEW

**Comment by rayrod2030** [comment]
test this please

**Comment by medidata** [comment]
Test failed. :poop:
Refer to this link for build results: http://ec2-54-221-39-184.compute-1.amazonaws.com:8080/job/chronos_cookbook_test/1/

**Comment by rayrod2030** [comment]
test this please

**Comment by medidata** [comment]
Test PASSED! :thumbsup:
Refer to this link for build results: http://ec2-54-221-39-184.compute-1.amazonaws.com:8080/job/chronos_cookbook_test/4/

**Comment by medidata** [comment]
Test PASSED! :thumbsup:
Refer to this link for build results: http://ec2-54-221-39-184.compute-1.amazonaws.com:8080/job/chronos_cookbook_test/5/

File: .kitchen.ec2.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<%  # This file contains ERB-interpreted YAML directives for testkitchen.
+    # Secrets are read from the local execution ENV into this file, and
+    # passed to the Chef tests via node attributes applied to the test suites.
+
+# To keep the YAML as simple as possible, some values are computed here with defaults set so kitchen will run without
+# values being set in ENV - at least as far as listing or running on the vagrant driver
+AWS_ACCESS_KEY_ID = ENV['AWS_BLUE_ACCESS_KEY_ID'] ||'(ENTER_AWS_ACCESS_KEY_ID)'
```
Comment (harryw) [inline .kitchen.ec2.yml]: maybe these env vars should be something more generic for a public cookbook

**Comment by medidata** [comment]
Test PASSED! :thumbsup:
Refer to this link for build results: http://ec2-54-221-39-184.compute-1.amazonaws.com:8080/job/chronos_cookbook_test/6/

File: .kitchen.ec2.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+def yaml_section(hash, indentation)
+  YAML.dump(hash).sub(/\A---\s/, '').split("\n").map do |l|
+    (" " * indentation) + l
+  end.join("\n")
+end
+%>
+---
+driver:
+  name: ec2
+  aws_access_key_id: <%= ENV['AWS_BLUE_ACCESS_KEY_ID'] ||
```
Comment (harryw) [inline .kitchen.ec2.yml]: Same thing here and 2 lines below...

**Comment by medidata** [comment]
Test failed. :poop:
Refer to this link for build results: http://ec2-54-221-39-184.compute-1.amazonaws.com:8080/job/chronos_cookbook_test/7/

**Pull request merged**